styles/main.css

```
@@ -41,6 +52,65 @@ button.primary {
+  border-radius: 40px;
+  border: 1px solid var(--color-bluegray-30);
+  background: var(--color-bluegray-90);
+}
+
+.search-input {
+  flex: 1;
+  border: none;
+  background: transparent;
+  outline: none;
+
+  color: white;
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)